release-notes/release-notes57.md

```diff
@@ -49,15 +49,25 @@ The following enhancements were made for the {{site.productname}} 5.7 release.
 
 The following premium plugin updates were released alongside {{site.productname}} 5.7.
 
-### Premium Plugin Name X.Y.Z
+### Spell Checker Pro 2.3.0
 
-The {{site.productname}} 5.7 release includes an accompanying release of the **<<Premium Plugin Name>>** premium plugin.
+The {{site.productname}} 5.7 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
 
-**<<Premium Plugin Name>>** X.Y.Z provides the following improvements:
+**Spell Checker Pro** 2.3.0 provides the following enhancements:
 
-- <Description>
+### The `spellchecker_ignore_list` option now accepts arrays of words for specific languages
 
-For information on the <<Premium Plugin Name>> plugin, see: [<<Premium Plugin Name>> plugin]({{site.baseurl}}/plugins/<<Premium Plugin Name>>/).
+**Spell Checker Pro** 2.3.0 introduces enhancements to the `spellchecker_ignore_list` (formerly `spellchecker_whitelist`, see: [The `spellchecker_whitelist` option has been renamed](#thespellchecker_whitelistoptionhasbeenrenamed)).
+
+- It is now possible to specify arrays of words for specific languages to be ignored by Spell Checker Pro plugin using the `spellchecker_ignore_list` option.
+- A new `addIgnoredWords` API allows words to be ignored at runtime by the Spell Checker Pro plugin.
+
+For information on:
+
+- The `spellchecker_ignore_list` option, see: [Spell Checker Pro - `spellchecker_ignore_list`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#spellchecker_ignore_list).
+- The `addIgnoredWords` API, see: [Spell Checker Pro - `API`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#api).
+
+For information on the Spell Checker Pro plugin, see: [Spell Checker Pro plugin]({{site.baseurl}}/plugins/premium/tinymcespellchecker/).
```

Comment:
can you add this to the list above? looks weird hanging down here.

Reply:
The list above is prefixed with "for information on" and already contains two links to the plugin, so maybe we could just remove this?

Reply:
sure. go for it.